plugins/chain_plugin/chain_plugin.cpp

```diff
@@ -2073,7 +2073,8 @@ fc::variant read_only::get_block(const read_only::get_block_params& params) cons
    EOS_ASSERT( block, unknown_block_exception, "Could not find block: ${block}", ("block", params.block_num_or_id));
 
    fc::variant pretty_output;
-   abi_serializer::to_variant(*block, pretty_output, make_resolver(this, abi_serializer_max_time), abi_serializer_max_time);
+   auto abi_yield = abi_serializer::create_yield_function( abi_serializer_max_time );
```

Comment:
const

Reply:
Actually removed this and went with double invocation since a copy would be made anyway.